README.md

```diff
@@ -67,6 +66,7 @@ Here are all the samples which are deployed on Microsoft Azure and you can try i
 |9| Universal bots  | Teams catering bot demonstrating how to use Universal bots in Teams | [View](samples/bot-teams-catering/csharp) |  | |
 |10| Sequential workflow adaptive cards | Demonstrating on how to implement sequential flow, user specific view and upto date adaptive cards in bot. | [View][sequential#workflow#csharp] | [View][sequential#workflow#js]
 |11| Channel messages with RSC permissions | Demonstrating on how a bot can receive all channel messages with RSC without @mention. | [View][messageswithrsc#csharp] |[View][messageswithrsc#js]
+|12| Bot with SharePoint file viewer| This sample demos a bot which uploads file to SharePoint site and same files can be viewed in Teams file viewer. | [View][botwithsharepointfileviewer#csharp] |
```

Comment:
This sample demos a bot with capability to upload files to SharePoint site and same files can be viewed in Teams file viewer.